pkg/client/client.go

```diff
@@ -89,12 +89,12 @@ type Client struct {
 	observed        proto.StateObserved_Status
 	observedMessage string
 
-	ctx    context.Context
-	cancel context.CancelFunc
-	wg     sync.WaitGroup
-	client proto.ElasticAgentClient
-	cfgLock   sync.RWMutex
-	obsLock   sync.RWMutex
+	ctx     context.Context
```

Comment:
We probably should have a formatter check as part of CI 🤔

Reply:
https://github.com/elastic/elastic-agent-client/issues/6 yup :)